app/src/main/java/com/mobileproto/hireddit/hireddit/speech/WordToSpeech.java

```diff
@@ -41,6 +41,12 @@ public void speak(String toSpeak)
         }
     }
 
+    public void pause() {
+        if (mTtobj.isSpeaking()) {
+            mTtobj.stop();
+        }
+    }
+
     public void stop()
```

Comment:
How are pause and stop different? Seems like it makes sense to have only pause and not stop?